src/core_atmosphere/physics/mpas_atmphys_driver_radiation_lw.F

```diff
@@ -426,6 +432,12 @@ subroutine radiation_lw_from_MPAS(xtime_s,configs,mesh,state,time_lev,diag_physi
 
  end if
 
+ if (mpas_cpl%role_is(ROLE_INTEGRATE)) then
```

Comment:
Here we're using `role_is`, while what looks like the symmetric transfer on L.405 uses `role_includes`. Should the if-tests in both cases be the same?